mlir/include/mlir/Dialect/LLVMIR/NVVMOps.td

```diff
@@ -2317,7 +2296,7 @@ def NVVM_WgmmaMmaAsyncOp : NVVM_Op<"wgmma.mma_async",
 }
 
 //===----------------------------------------------------------------------===//
-// NVVM breakpoint Op
+// NVVM Miscellaneous instructions
```

Comment:
Yes, these are under the misc instructions section in the spec.

Thanks for moving the setmaxnreg Op